app/helpers/countries_helper.rb

```diff
@@ -38,7 +38,8 @@ def total_value(dataset)
   end
 
   def total_units(dataset)
-    total_units = dataset[:id] == 'coldcorals' ? 'observations' : 'km<sup>2</sup>'.html_safe
+    # total_units = dataset[:id] == 'coldcorals' ? 'observations' : 'km<sup>2</sup>'.html_safe
+    'km<sup>2</sup>'.html_safe if total_units = dataset[:id] == 'coldcorals'
```

Comment:
Forgot to delete the commented out code, for some reason coldcorals was set to use observations instead of km2. I've now deleted the entire method and put `km<sup>2</sup>` in the actual view as there is now no need for a conditional.